integration/auth/user_mgt_test.go

```diff
@@ -434,7 +434,9 @@ func TestCreateUserMFA(t *testing.T) {
 	tc.MFASettings(auth.MultiFactorSettings{
 		EnrolledFactors: []*auth.MultiFactorInfo{
 			{
-				PhoneNumber: "+11234567890",
```

Comment:
Same as below. We should still keep this test for backward compatibility and add a new test for `PhoneMultiFactorInfo`